lib/utils.php

```diff
@@ -255,4 +255,12 @@ function mozilla_redirect_admin() {
     }
 }
 
+function mozilla_verify_url($url) {
```

Comment:
Try to keep this type of notation for embedding logic in markup.  If it's straight PHP code stick to the convention of using brackets to denote blocks.

Instead of
` 
if(conditional):
   // code
endif;
`
use 

`
if(conditional) {
// code
}
` 
when strictly using php that is not embedded within markup.